lnbits/core/tasks.py

```diff
@@ -51,7 +51,7 @@ async def killswitch_task():
                                 "Switching to VoidWallet. Killswitch triggered."
                             )
                             await switch_to_voidwallet()
-                except (httpx.ConnectError, httpx.RequestError):
+                except (httpx.RequestError, httpx.HTTPStatusError):
```

Comment:
Is `httpx.ConnectError` not ever thrown?
Can we just catch `Error` and log the actual error?
Is there a case when the error should bubble up?

Reply:
ConnectError is capture bei Request error look at the exception tree here. it was redundant. https://www.python-httpx.org/exceptions/

Reply:
statuserror  is triggerbe by `raise_for_status()` and is raised if the response has a bad status code